reddit_liveupdate/controllers.py

```diff
@@ -1090,3 +1092,38 @@ def GET_mediaembed(self, liveupdate, embed_index):
         }
 
         return pages.LiveUpdateMediaEmbedBody(**args).render()
+
+
+@add_controller
+class LiveUpdateAdminController(RedditController):
```

Comment:
I was going to inject it into the menu separately, since I didn't really see a way to inject it in from here.